client/my-sites/site-settings/jetpack-site-stats.jsx

```diff
@@ -171,6 +172,10 @@ class JetpackSiteStats extends Component {
 						}
 					</FormFieldset>
 				</FoldableCard>
+
+				<CompactCard href={ '/stats/day/' + siteId }>
```

Comment:
Should insights be the default view instead? There's a utility that figures out where should the My Sites nav item send you to.